MPDroid/src/main/java/com/namelessdev/mpdroid/SearchActivity.java

```diff
@@ -188,25 +188,43 @@ protected void asyncUpdate() {
         final String finalSearch = mSearchKeywords.toLowerCase();
 
         List<Music> arrayMusic = null;
+        List<Music> arrayMusicFiles = null;
 
         try {
             arrayMusic = mApp.oMPDAsyncHelper.oMPD.search("any", finalSearch);
         } catch (final IOException | MPDException e) {
             Log.e(TAG, "MPD search failure.", e);
-
         }
 
         if (arrayMusic == null) {
             return;
```

Comment:
Thats right, but i thought arrayMusic is only null if an error occured (exception catched) and i don't wanted to do another search in that case. arrayMusic should be an empty list if nothing was found.